docs/protocol.html

```diff
@@ -143,7 +138,13 @@ <h5 class="anchor-heading"><a id="api_versions" class="anchor-link"></a><a href=
         Kafka protocol requests may take place on an SSL listener before the SSL handshake is finished). If this is considered to
         leak information about the broker version a workaround is to use SSL with client authentication which is performed at an
         earlier stage of the connection where the <code>ApiVersionRequest</code> is not available. Also, note that broker versions older
-        than 0.10.0.0 do not support this API and will either ignore the request or close connection in response to the request.</li>
+        than 0.10.0.0 do not support this API and will either ignore the request or close connection in response to the request. Also
```

Comment:
maybe better to extract it into it's own paragraph or section about fallback behaviour and link from here?